### ISSUE TITLE
Disappearing vertical scrollbar

### ISSUE THREAD

**Issue by hcsfred** [SpinyOwl/legui#107]
In a ScrollablePanel, if the container height is equal to the panel height and a horizontal mouse drag/scroll is performed then the vertical scrollbar will disappear.

**Comment by ShchAlexander** [comment]
Could you please provide an example of your ScrollablePanel creation and any style changes you applied to it, so I can reproduce the issue?

**Comment by hcsfred** [comment]
``` 
modelPanel = ScrollablePanel()
val width = 160f
modelPanel.style.maxWidth = width
modelPanel.container.size.x = width

modelPanel.style.setMargin(79f, 0f, 6f, 6f)
modelPanel.style.position = Style.PositionType.RELATIVE
modelPanel.style.flexStyle.flexGrow = 1
```

https://i.imgur.com/d6RQxI3.gif

**Comment by hcsfred** [comment]
Fixed in #109
